### ISSUE TITLE
Error in reproducing results

### ISSUE THREAD

**Issue by kietdoan** [yuchen2199/CSBrain#3]
First of all. thank you for sharing this great work. 
I have some questions in reproducing the results. For example, in BCI2a dataset, the bash file you provide run with weight decay 0.01, but in the appendix, you state 0.05. I try both these values but the performance degrades remarkably. What is the correct hyperparameters for this task to reproduce the result as provided in the pth_downtasks checkpoint.
Best regards,

**Comment by yuchen2199** [comment]
Both 0.01 and 0.05 weight decay values achieved similar performance in our experiments. We recommend starting with 0.01. Please ensure you have performed the correct data preprocessing and correctly loaded the pre-trained weights. Here are the detailed parameters and results from one of our successful runs for your reference:

Test Evaluation: acc: 0.57726, kappa: 0.43634, f1: 0.56665
seed: 42 lr: 0.0001 weight_decay: 0.01 dropout: 0.3

Note that performance can be influenced by hardware, software environment, and hyperparameters. To achieve the best and stable performance in your environment, you may need to dynamically adjust hyperparameters.
